src/js/disability-benefits/526EZ/components/ReviewCardField.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+
+import {
+  getDefaultFormState,
+  getDefaultRegistry
+} from '@department-of-veterans-affairs/react-jsonschema-form/lib/utils';
+
+import { errorSchemaIsValid } from '../../../common/schemaform/validation';
+
+import set from '../../../../platform/utilities/data/set';
+import get from '../../../../platform/utilities/data/get';
+
+
+/**
+ * Displays a review card if the information inside is valid.
+ *
+ * For use on a schema of type 'object' or 'array'.
+ * Intended to wrap objects or arrays to avoid duplicate functionality here.
+ */
+export default class ReviewCardField extends React.Component {
+  static defaultProps = {
+    uiSchema: {},
+    errorSchema: {},
+    idSchema: {},
+    registry: getDefaultRegistry(),
+    required: false,
+    disabled: false,
+    readonly: false,
+  }
+
+
+  constructor(props) {
+    super(props);
+
+    // Throw an error if there’s no viewComponent (should be React component)
+    if (typeof get('ui:options.viewComponent', this.props.uiSchema) !== 'function') {
+      throw new Error(`No viewComponent found in uiSchema for ReviewCardField ${this.props.idSchema.$id}.`);
+    }
+
+    if (!['object', 'array'].includes(this.props.schema.type)) {
+      throw new Error(`Unknown schema type in ReviewCardField: ${this.props.schema.type}`);
```

Comment:
Might suggest hinting at the allowed schema types instead:
`Expected schema type of 'array' || 'object' but got ${this.props.schema.type} instead`